schedule.md

```diff
@@ -53,16 +53,14 @@ The following schedule is subject to change, and many slide links are not yet ac
 </div>
 <div class="week">
 
-## Week 2 (October 4-6)
+## Week 2 (October 9-11)
 
 ### Topics
 
 - Scaling Web Applications ([slides](/slides/2022/05_scaling_web_applications/))
```

Comment:
Probably want to drop all the links to slides and update them once the class/content for that week is up.